Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -52,6 +52,11 @@ namespace Hazel {
 			glfwSetErrorCallback(GLFWErrorCallback);
 		}
 
+
+		glfwWindowHint(GLFW_CONTEXT_VERSION_MAJOR, 4);
+		glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 5);
+		glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE);
```

Comment:
This file does handle GLFW and adding OpenGL requirements makes sense but maybe it'd be more appropriate in an OpenGL project file?  We know that the specification of Hazel is OpenGL version 4.5 yet Cherno didn't add OpenGL functions to enforce this requirement.

Reply:
I don't think this is bad, however I think it should add a check around it to check the rendering api to make sure openGL is being used?

I also think this needs to move down a but, such it is part of the `HZ_PROFILE_SCOPE("glfwCreateWindow");` ?